editor.planx.uk/src/@planx/components/FileUploadAndLabel/Public.tsx

```diff
@@ -89,6 +89,7 @@ function Component(props: Props) {
   }, []);
 
   const [slots, setSlots] = useState<FileUploadSlot[]>([]);
+  const orderByMostRecentlyUploaded = true;
```

Comment:
I'm not sure about this pattern to be honest but I was reluctant to directly change the order of the `slots` and I liked the idea of having one variable at the parent level to determine the order.

Reply:
I'm totally on board with just wanting to define order in one high-level place, but personally I think a boolean prop to control this is at the component level is a bit unnecessary and would never be changed? 

When user research makes a suggestion for a design change based on feedback, and it's not a style attribute that needs to be exposed to editors, then we should just full out adopt it in my opinion and no need to add the internal complexity of making it configurable.

Reply:
That's a very good point @jessicamcinchak I think I still have the mindset of my last job where we were generally always A/B testing and tended to follow this sort of pattern. I'll get this update 😊